test/ComposableStable.integration.spec.ts

```diff
@@ -88,14 +88,70 @@ const testPool: SubgraphPoolBase = {
     amp: '1472',
 };
 
+// 4pool chainlink - example containing tokens with less than 18 decimals
+const testPool1: SubgraphPoolBase = {
```

Comment:
Unfortunately both the pools with <18 decimals seem to have broken priceRate providers which causes the query to revert. Probably just remove this for now.